### PR TITLE
Add sysadmin guide to the documentation

### PR DESCRIPTION
**Pull request by jaegeral**
Updating the Documentation around sysadmin stuff.

First iteration for

https://github.com/google/timesketch/issues/1411

### CODE REVIEW

File: docs/sysadmin/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+- Any output on Chrome console?
+- Try a different device / clear caches from browser
+- Look in nginx / webserver logs
+- Look in Timesketch / celery worker logs
+- Look in ES logs
+- Look in postgres logs
+
+## UX issues
+
+To troubleshoot UX issues, the first look should go to the Chrome developer console and look for any output / errors.
```
Comment (berggren) [inline docs/sysadmin/troubleshooting.md:15]:
```suggestion
To troubleshoot web frontend issues, the first look should go to the Chrome developer console and look for any output / errors.
```

File: docs/sysadmin/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+To raise issues related to UX, please provide the following:
```
Comment (berggren) [inline docs/sysadmin/troubleshooting.md:17]:
```suggestion
To raise issues related to the web frontend, please provide the following:
```

File: docs/sysadmin/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+### Timesketch / celery worker
+
+See your console output if you started the workers with:
+
+```shell
+docker exec -it $CONTAINER_ID celery -A timesketch.lib.tasks worker --loglevel=debug
+```
+
+### ElasticSearch
```
Comment (berggren) [inline docs/sysadmin/troubleshooting.md:108]:
```suggestion
### Elasticsearch
```

File: docs/sysadmin/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+```shell
+2020-08-19 14:40:48,912 [ERROR] (MainProcess) PID:568 <pinfo_tool> Format of storage file: $FILENAME not supported
+```
+
+## logs
+
+All of those are subject to change depending on your operating system.
+
+### nginx / webserver
```
Comment (berggren) [inline docs/sysadmin/troubleshooting.md:76]:
```suggestion
### Nginx / webserver
```

File: docs/sysadmin/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+pinfo.py $FILENAME
+```
+
+Should give the following error message:
+
+```shell
+2020-08-19 14:40:48,912 [ERROR] (MainProcess) PID:568 <pinfo_tool> Format of storage file: $FILENAME not supported
+```
+
+## logs
```
Comment (berggren) [inline docs/sysadmin/troubleshooting.md:72]:
```suggestion
## Logs
```

File: docs/sysadmin/troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+docker container list
+```
+
+Which will show something like
+
+```shell
+b756f334d281   us-docker.pkg.dev/osdfir-registry/timesketch/dev:latest        "/docker-entrypoint.…"   8 days ago   Up 2 days   127.0.0.1:5000->5000/tcp             timesketch-dev
+7768635b4798   us-docker.pkg.dev/osdfir-registry/timesketch/notebook:latest   "jupyter notebook"       8 days ago   Up 2 days   127.0.0.1:8844->8844/tcp, 8899/tcp   notebook
+6d2e7da9453c   justwatch/elasticsearch_exporter:1.1.0                         "/bin/elasticsearch_…"   8 days ago   Up 2 days   9114/tcp                             es-metrics-exporter
+78443764330b   grafana/grafana:latest                                         "/run.sh"                8 days ago   Up 2 days   127.0.0.1:3000->3000/tcp             grafana
```
Comment (berggren) [inline docs/sysadmin/troubleshooting.md:42]: This is part of the dev container. We have not enabled the metrics on the prod deployment yet. Maybe we should adjust the ouput here in the meantime?

**Review by berggren — DISMISSED** [review]

File: docs/sysadmin/troubleshooting.md (hunk 6 above)
Comment (jaegeral) [reply docs/sysadmin/troubleshooting.md]: yes good point, removed metrics part

File: docs/sysadmin/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+docker exec -it $CONTAINER_ID gunicorn --reload -b 0.0.0.0:5000 --log-file - --timeout 600 timesketch.wsgi:application
+```
+
+Or run the following where ```$CONTAINER_ID``` is the ID from your ```timesketch-dev```or ```timesketch``` docker image
+
+```shell
+docker logs $CONTAINER_ID
+```
+
+### Timesketch / celery worker
```
Comment (tomchop) [inline docs/sysadmin/troubleshooting.md:98]:
```suggestion
### Timesketch / Celery worker
```

File: docs/sysadmin/troubleshooting.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+docker exec -it $CONTAINER_ID celery -A timesketch.lib.tasks worker --loglevel=debug
+```
+
+### Elasticsearch
+
+```shell
+/var/log/elasticsearch
+```
+
+Or run the following where ```$CONTAINER_ID``` is the ID from your ```elasticsearch``` docker image
```
Comment (tomchop) [inline docs/sysadmin/troubleshooting.md:112]:
```suggestion
Or run the following where `$CONTAINER_ID` is the ID from your `elasticsearch` Docker image
```

File: docs/sysadmin/troubleshooting.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+docker logs $CONTAINER_ID
+```
+
+### Postgress
+
+```shell
+/var/log/postgresql/
+```
+
+Or run the following where ```$CONTAINER_ID``` is the ID from your ```postgres``` docker image
```
Comment (tomchop) [inline docs/sysadmin/troubleshooting.md:124]:
```suggestion
Or run the following where `$CONTAINER_ID` is the ID from your `postgres` Docker image
```

**Review by tomchop — CHANGES_REQUESTED** [review]

File: docs/sysadmin/troubleshooting.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+Good starter is to run the following grep:
+
+```shell
+grep "http_code:500"  /var/log/nginx/access.log #to get all Error 500
+```
+
+If you started the webserver with docker, look in the console where you started:
+
+```shell
+docker exec -it $CONTAINER_ID gunicorn --reload -b 0.0.0.0:5000 --log-file - --timeout 600 timesketch.wsgi:application
```
[the remark this answers is not in the document]
Comment (jaegeral) [reply docs/sysadmin/troubleshooting.md:89]: good catch

**Review by tomchop — APPROVED** [review]

**Pull request merged**